### DISCUSSION TITLE
Specifying a project's HTML output directory

### DISCUSSION THREAD

**Discussion by dciomics** [bdtc/lwarp#27] [Ideas]
Apologies if I've missed something obvious, but it would be very convenient to have an lwarp parameter that would specify the directory to be used for all output (HTML, images, css, etc.) that are required to be a DocumentRoot in a web server. Currently I have to use scripts to separate (move or copy) the generated output files and required support files from the source and auxiliary files in the project directory into a separate "clean" output directory. With this functionality built into lwarp, one could simply copy the project's output directory to a web server and be ready to go.

**Comment by bdtc** [comment]
This functionality will eventually be useful for direct EPUB generation as well.  It's on the todo "some day" list.
